docs/usage.md

```diff
@@ -0,0 +1,423 @@
+        <file src="tools/**/*" target="tools" />
+    </files>
+</package>
+```
+
+**Key points:**
+
+- Set a unique `<id>` for your package
+- Use semantic versioning for `<version>`
+- Include `<packageType name="DotnetTool" />` to mark this as a .NET tool
+- Adjust file paths in the `<files>` section to match your structure
+
+Refer to the [.nuspec File Reference](https://learn.microsoft.com/en-us/nuget/reference/nuspec) for more details.
+
+### 2. Create DotnetToolSettings.xml
+
+Create a `DotnetToolSettings.xml` file in each framework folder (e.g., `tools/net8.0/any/`):
+
+```xml
+<?xml version="1.0" encoding="utf-8"?>
+<DotNetCliTool Version="1">
+  <Commands>
+    <Command Name="my-tool" EntryPoint="DemaConsulting.DotnetToolWrapper.dll" Runner="dotnet" />
+  </Commands>
+</DotNetCliTool>
+```
+
+**Key points:**
+
+- Replace `my-tool` with the command name users will type to run your tool
+- Keep `EntryPoint` as `DemaConsulting.DotnetToolWrapper.dll`
+- Keep `Runner` as `dotnet`
+
+### 3. Create DotnetToolWrapper.json
+
+Create a `DotnetToolWrapper.json` file in each framework folder that maps platform identifiers to executables:
+
+```json
+{
+  "win-x64": {
+    "program": "win-x64/my-tool.exe"
+  },
+  "win-x86": {
+    "program": "win-x86/my-tool.exe"
+  },
+  "linux-x64": {
+    "program": "linux-x64/my-tool"
+  },
+  "linux-arm64": {
+    "program": "linux-arm64/my-tool"
+  },
+  "osx-x64": {
+    "program": "osx-x64/my-tool"
+  },
+  "osx-arm64": {
+    "program": "osx-arm64/my-tool"
+  }
+}
+```
+
+**Platform Identifiers:**
+
+The platform identifier format is `{os}-{architecture}`:
+
+**Supported Operating Systems:**
+
+- `win` - Windows
+- `linux` - Linux
+- `freebsd` - FreeBSD
+- `osx` - macOS
+
+**Supported Architectures:**
+
+- `x86` - 32-bit Intel/AMD
+- `x64` - 64-bit Intel/AMD
+- `arm` - 32-bit ARM
+- `arm64` - 64-bit ARM (Apple Silicon, etc.)
+- `wasm` - WebAssembly
+- `s390x` - IBM System z
+
+**Key points:**
+
+- Only include platforms you support
+- Paths are relative to the DotnetToolWrapper.json file location
+- Environment variables in paths will be expanded (e.g., `%USERPROFILE%`, `$HOME`)
+- The wrapper will automatically select the correct executable based on the runtime platform
+
+### 4. Copy DotnetToolWrapper Files
+
+For each framework version you're supporting, copy the following files from the DotnetToolWrapper release
+into your `tools/{framework}/any/` folder:
+
+- `DemaConsulting.DotnetToolWrapper.dll`
+- `DemaConsulting.DotnetToolWrapper.deps.json`
+- `DemaConsulting.DotnetToolWrapper.runtimeconfig.json`
+
+You can obtain these files by:
+
+1. Downloading from the [releases page](https://github.com/demaconsulting/DotnetToolWrapper/releases)
+2. Building from source (see [Building from Source](#building-from-source))
+
+### 5. Add Your Native Executables
+
+Place your native executables in subdirectories matching the platform identifiers. For example:
+
+```text
+tools/net8.0/any/
```

Comment:
Update the folder structure to indicate the folders are to be placed at the root of the package.